src/java/fr/paris/lutece/plugins/identitymediation/web/IdentityDuplicateJspBean.java

```diff
@@ -663,6 +680,31 @@ public String doCreateIdentityMergeTask(final HttpServletRequest request) throws
         return getResolveDuplicates( request );
     }
 
+    private void verifyExistingTasks( ) throws IdentityStoreException
+    {
+        _lastNotif = null;
+        final IdentityTaskListGetResponse response =
+                _serviceIdentity.getIdentityTaskList(_identityToKeep.getCustomerId(), IdentityResourceType.CUID.name( ), _currentClientCode, this.buildAgentAuthor( ) );
+        if ( response != null )
+        {
+            final List<IdentityTaskDto> tasks = response.getTasks();
+            if ( tasks != null && !tasks.isEmpty( ) )
+            {
+                final String taskType = _identityToKeep.isMonParisActive() && _identityToMerge.isMonParisActive() ?
+                                        IdentityTaskType.ACCOUNT_MERGE_REQUEST.name() : IdentityTaskType.ACCOUNT_IDENTITY_MERGE_REQUEST.name();
```

Comment:
The ACCOUNT_IDENTITY_MERGE_REQUEST type only applies to a connected primary identity and an unconnected secondary identity.
This test can return an ACCOUNT_IDENTITY_MERGE_REQUEST type even if both identities are not connected; therefore it should be safer to restrict the test.